docs/resources/aws_redshift_cluster.md

```diff
@@ -0,0 +1,82 @@
+---
+title: About the aws_redshift_cluster Resource
+---
+
+# aws\_redshift\_cluster
+
+Use the `aws_redshift_cluster` InSpec audit resource to test detailed properties of an individual Redshift cluster.
+
+
+## Syntax
+
+An `aws_redshift_cluster` resource block uses resource parameters to search for an Redshift cluster, and then tests that Redshift cluster.  If no Redshift clusters match, no error is raised, but the `exists` matcher will return `false` and all properties will be `nil`.  If more than one Redshift cluster matches (due to vague search parameters), an error is raised.
+
+    describe aws_redshift_cluster('test-cluster-id') do
+      it { should exist }
+    end
+
+    # Can also use hash syntax
+    describe aws_redshift_cluster(cluster_identifier: 'test-cluster-id') do
+      it { should exist }
+    end
+
+#### Parameters
+
+##### cluster\_identifier _(required)_
+
+This resource accepts a single parameter, the user-supplied cluster identifier. This parameter isn't case-sensitive.
+This can be passed either as a string or as a `cluster_identifier: 'value'` key-value entry in a hash.
+
+See also the [AWS documentation on Redshift cluster](https://docs.aws.amazon.com/AmazonRedshift/latest/AuroraUserGuide/Aurora.Overview.html).
```

Comment:
This link is bad. Is https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Overview.html the correct link?